lib/makeserver.js

```diff
@@ -100,6 +149,24 @@ module.exports = function(options) {
         });
     }
 
+    app.get('*', function (req, res, next) {
+      // redirect stock standard help page paths paths
+      const redirects = [
+        { oldPath: "/about.html", newPath: "/about" },
+        { oldPath: "/help/help.html", newPath: "/about/help" },
+        { oldPath: "/help/faq.html", newPath: "/about/faq" },
+        { oldPath: "/help/privacy.html", newPath: "/about/privacy" },
+      ];
+      redirects.forEach(redirect => {
+        if (redirect.oldPath === req.path) res.redirect(301, redirect.newPath);
+      });
+      next();
+      // 301 any other about pages before they get hit if they still exist (?)
+      // if (req.path.indexOf("about") !== -1 && req.path.indexOf(".html") !== -1) {
+      //   res.redirect(301, `${req.path.slice(0, -5)}`);
+      // }
```

Comment:
Is the "301 common historical about/help `.html` routes" required at the moment?

Reply:
No but leaving it there so that any maps that get migrated, will redirect to the right path